projects/experimental/directives/button-group/button-group.style.less

```diff
@@ -0,0 +1,72 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiButtonGroup] {
+    display: flex;
+    border-radius: 1.5rem;
+    justify-content: center;
+    overflow: hidden;
+
+    > button {
+        .button-base();
+
+        position: relative;
+        flex-grow: 1;
+        flex-basis: 0;
+        border-radius: 0;
+        flex-direction: column;
+        padding: 1.125rem 0.25rem 1.125rem 0.25rem;
+        gap: 0.5rem;
+        height: auto;
+        font-size: 0.8125rem;
```

Comment:
Should probably be `var(--tui-font-text-s)`

Reply:
fixed